apps/meteor/app/api/server/lib/messages.ts

```diff
@@ -84,10 +83,6 @@ export async function findStarredMessages({
 }
 
 export async function findSnippetedMessageById({ uid, messageId }: { uid: string; messageId: string }): Promise<IMessage> {
-	if (!(await getValue('Message_AllowSnippeting'))) {
```

Comment:
Are we removing the hability to snippet a message?

Reply:
Ok, so you should remove the whole snippet logics, apis, etc, then.

Reply:
Because as you did you are just removing the setting that allow disabling the feature, and the API been there it will always be allowed now even if we don't have a UI to see.